scripts/ci/publish.ts

```diff
@@ -583,8 +592,8 @@ async function publish() {
       }
       const passing = await areEcosystemTestsPassing(tagForEcosystemTestsCheck)
       if (!passing && process.env.SKIP_ECOSYSTEMTESTS_CHECK !== 'true') {
-        throw new Error(`We can't release, as the ecosystem-tests are not passing for the ${tag} npm tag!
-Check them out at https://github.com/prisma/ecosystem-tests/actions?query=workflow%3Atest+branch%3A${tag}`)
+        throw new Error(`We can't release, as the ecosystem-tests are not passing for the ${tag ?? 'N/A'} npm tag!
```

Comment:
What do we even check in this script if we don't have a tag to map to the branch?